PowerEditor/src/NppDarkMode.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <Windows.h>
```

Comment:
Try to avoid including `<Windows.h>` especially in headers. This header is large and it affects the compile time.
If you only need it for `COLORREF` and `HBRUSH` then include `<windef.h>` and `<wingdi.h>` respectively.